apps/charterafrica/src/lib/ecosystem/collections.js

```diff
@@ -0,0 +1,34 @@
+import {
```

Comment:
Why is this here? Is it useful anywhere else besides the /v1/apis/?

Reply:
@kilemensi at the moment this is only useful in ecosystem since the only processed collections are tools, organisations and contributors